client/src/components/Ticketing/ticketingmanager/showings/showing/showing.tsx

```diff
@@ -79,7 +80,7 @@ const Showing = () => {
   if (eventData === undefined) return <p>Whoops! Event not found</p>;
   const {title, description, imageurl, tickets} = eventData;
   return (
-    <div className='w-full h-screen overflow-x-hidden absolute bg-cover' style={{backgroundImage: `url(${imageurl})`}}>
+    <div className='w-full h-screen overflow-x-hidden absolute bg-cover' style={{backgroundImage: `url(${getImageDefault(imageurl)}),url(${getImageDefault()})`}}>
```

Comment:
Just a curious question, but why use `getImageDefault()` twice?

Reply:
the second call returns the default image as a fall back incase the first url is broken.